app/src/main/res/values/strings.xml

```diff
@@ -470,4 +470,12 @@ Upload your first media by tapping on the add button.</string>
   <string name="image_chooser_title">Choose Images to upload</string>
 
   <string name="please_wait">Please wait…</string>
+
+  <string name="nominating_for_Deletion">Nominating for Deletion</string>
+  <string name="success">Success</string>
+  <string name="successfully_nominated">Successfully nominated</string>
+  <string name="for_deletion">for deletion</string>
```

Comment:
Please read https://github.com/commons-app/apps-android-commons/issues/2498 and fix accordingly.